src/portal/lib/src/service/scanning.service.ts

```diff
@@ -106,17 +109,22 @@ export class ScanningResultDefaultService extends ScanningResultService {
     tagId: string,
     queryParams?: RequestQueryParams
   ):
-    | Observable<VulnerabilityItem[]> {
+    | Observable<any> {
     if (!repoName || repoName.trim() === "" || !tagId || tagId.trim() === "") {
       return observableThrowError("Bad argument");
     }
 
+    let httpOptions = buildHttpRequestOptions(queryParams)
+    let requestHeaders = httpOptions.headers as HttpHeaders
+    // Change the accept header to the supported report mime types
+    httpOptions.headers = requestHeaders.set("Accept", DEFAULT_REPORT_MIME_TYPE)
+
     return this.http
       .get(
-        `${this._baseUrl}/${repoName}/tags/${tagId}/vulnerability/details`,
-        buildHttpRequestOptions(queryParams)
+        `${this._baseUrl}/${repoName}/tags/${tagId}/scan`,
+        httpOptions
       )
-      .pipe(map(response => response as VulnerabilityItem[])
+      .pipe(map(response => response as any)
```

Comment:
Why modified VulnerabilityItem[] to any? If you know the type, you can use "|" to identify the types.

Reply:
fixed